finstmt/findata/database.py

```diff
@@ -134,3 +127,39 @@ def get_sympy_subs_dict(self, t_offset: int = 0) -> Dict[IndexedBase, float]:
                 self.prior_statement.get_sympy_subs_dict(t_offset=t_offset - 1)
             )
         return subs_dict
+
+    #### NEW
+
+    t = symbols("t", cls=Idx)
+
+    # Get item even if attribute exists
+    def __getattribute__(self, key: str):
+        # print("FinDataBase.__getattribute__", key)
```

Comment:
Seems like you might want to use `__getattr__` here? It is better when you just want to have it triggered when an item is not already directly assigned to an attribute. Its seems like you are filtering out those cases explicitly here instead.

Also in general it feels like we are bending over backwards a bit to make the dataclass work. 

I could imagine something like a `StatementItem` class that contains the config as well as the current value. Then we could just store a list of `StatementItem`s directly on the parent class or have this class just be basically a list of `StatementItem`s.

Reply:
done in the latest commit. 
removed dataclass on the financial statements.